docs/queries/select.mdx

```diff
@@ -137,6 +137,11 @@ export const Pages: CollectionConfig<'pages'> = {
 }
 ```
 
+<Banner type="warning">
+  <strong>Important:</strong>
+  When using `defaultPopulate` on collections with a [Upload](/docs/fields/upload) field, it is important to specify `filename: true` and `url: true`, otherwise Payload will not be able to fetch the correct image URL and populate the URL field in your queries, instead returning `url: null`.
+</Banner>
+
```

Comment:
I think
> on collections with a [Upload](/docs/fields/upload) field,

Is confusing as you probably mean using it on collections that have enabled uploads https://payloadcms.com/docs/upload/overview, not on upload fields.

Reply:
Yes that is what I meant, messed up the wording.